### DISCUSSION TITLE
汚いことばづかいを減らせないか

### DISCUSSION THREAD

**Discussion by kuboaki** [increments/qiita-discussions#211] [要望・アイデア・不具合]
投稿の中には、口にするのがはばかられることばを使ったり、煽るような言い回しをしていたり、内容が把握できないタイトルを使っているものがあります。

- 排泄物の呼称を使って「XXコード」という
- 推理小説のタイトルや会話のような、記事の意図がわからないタイトル

これに関連して、Facebookなどへのレコメンドや、人気記事紹介のメールでも、これらが選ばれていることも気になっています。この場合、投稿者ではなく運営側が上記のことは気にしていないのかと感じます。
また、こちらの都合によらず、タイムラインや受信リストの中で目についてしまいます。

どんなものが該当するのかについて、きちんとした基準を設けるのは難しいのかもしれません。
ですが、ある程度でも、記事のタイトル、本文、外部へのフィードなどで、こういったことを抑制できないものでしょうか。あるいは、好ましくない記事をフィードから除外する方法は提供できないものでしょうか。

誤解しないでいただきたいのは、昨今話題になっている差別に通じる用語の排斥とは別の問題ということです。
もちろん、大多数の記事はそういった問題はなく、たいへんありがたい存在です。

**Comment by tanaka-asami415** [comment]
@kuboaki 
ご意見をいただきありがとうございます🙏

誹謗中傷や読み手にわかりやすい記事を書くことは、ガイドラインにて規定しておりますが
基準が難しく、最終的にはユーザーさまに判断を委ねております🙏
https://help.qiita.com/ja/articles/qiita-community-guideline

「XXコード」などは、線引きが難しいことと、運営が個人の作成した記事に介入しすぎることは、
ユーザーのみなさまも好ましく感じられないと考えるため、現運用となっています🙏:sweat_drops:

> 好ましくない記事をフィードから除外する方法は提供できないものでしょうか。

ミュート機能がありますので、こちらをご活用いただけると幸いです。
https://help.qiita.com/ja/articles/qiita-mute

Facebookなどへのレコメンドや、人気記事紹介のメールについても、どこまで介入するのかが難しく
恣意的に選ばず、機械的に選別しております🙏

現時点では、ご要望に応えられず申し訳ありませんが、ご理解いただけますと幸いです。
また、もし投稿がコミュニティガイドラインなどに違反してましたら、お手数ですが通報をしていただけると幸いです:bow:

**Comment by kuboaki** [comment]
サポート担当者様、

ご回答ありがとうございます。
運営側の事情や対応には難しさがあることは理解できます。
問題の認識を共有できたことで、いったんの納得を得ました。

引き続き、コミュニティの健全な維持、発展にご尽力いただけますよう、お願い申し上げます。
当方も、記載内容に配慮を継続したいと思います。

-- kuboaki


2022年5月2日(月) 15:35 tanaka asami ***@***.***>:
